File: June-LeetCoding-Challenge/29-Unique-Paths/Unique-Paths.cs
public class Solution {
    public int UniquePaths(int m, int n) {
        int[,] dp = new int[m, n];
        for(int i=0; i<m; i++){
            for(int j=0; j<n; j++){
                if(i==0 || j==0){
                    dp[i,j] =1;
                }else {
                    dp[i,j] = dp[i,j-1] + dp[i-1,j];
                }               
            }
        }
        return dp[m-1,n-1];
    }
}

File: June-LeetCoding-Challenge/29-Unique-Paths/Unique-Paths.java
class Solution {
    public int uniquePaths(int width, int height) {
        int[][] grid = new int[height][width];

        for (int row = 0; row < height; row++) {
            for (int col = 0; col < width; col++) {
                if (row == 0 || col == 0) {
                    grid[row][col] = 1;
                } else {
                    grid[row][col] = grid[row - 1][col] + grid[row][col - 1];
                }
            }
        }
        return grid[height - 1][width - 1];
    }
}
